src/test/java/blackjack/domain/card/CardTest.java

```diff
@@ -0,0 +1,19 @@
+package blackjack.domain.card;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Objects;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class CardTest {
+
+    @Test
+    @DisplayName("equals & hashCode 재정의 테스트")
+    void equalsTest() {
+        Card card = new Card(CardShape.HEART, CardNumber.KING);
+
+        assertThat(card).isEqualTo(new Card(CardShape.HEART, CardNumber.KING));
+        assertThat(Objects.hash(card)).isEqualTo(Objects.hash(new Card(CardShape.HEART, CardNumber.KING)));
+    }
```

Comment:
나의 "equals & hashCode 재정의 테스트" 하는 방법 제안!
(조금 과하게 이것저것 검증 넣었으니까 적당히 필요한 것만 챙기면 될 듯)

```suggestion
    @Test
    @DisplayName("equals & hashCode 재정의 테스트")
    void equalsTest() {
        Card card1 = new Card(CardShape.HEART, CardNumber.KING);
        Card card2 = new Card(CardShape.HEART, CardNumber.KING);
        Card differentCard = new Card(CardShape.HEART, CardNumber.QUEEN);

        assertThat(card1).isEqualTo(card1)
                .isEqualTo(card2)
                .isNotEqualTo(differentCard)
                .hasSameHashCodeAs(card2);
    }
```